apps/components-e2e/src/components/chart/pie-chart/pie-chart.e2e.ts

```diff
@@ -49,6 +49,14 @@ test('should get the correct overlay values when hovering on the different pies'
     .expect(tooltip.textContent)
     .match(/15/)
     .hover(pieChart, { speed: 0.3, offsetX: 620, offsetY: 140 })
+    .wait(1000)
```

Comment:
if we reduce the speed to 0.1 for hovering I think we don"t need the wait here, or reduce the wait to 300 at least